src/compiler/commandLineParser.ts

```diff
@@ -1499,6 +1496,14 @@ namespace ts {
         }
     }
 
+    export function getErrorForNoInputFiles({ includeSpecs, excludeSpecs }: ConfigFileSpecs, configFileName?: string) {

```

Comment:
May be i should type configFileName as string | undefined as it could be undefined as it is also used through parseJsonConfigFileContent which is exposed api and has configFileName as optional parameter